sqlmesh/core/snapshot/definition.py

```diff
@@ -42,7 +42,8 @@
 
 Interval = t.Tuple[int, int]
 Intervals = t.List[Interval]
-SnapshotNode = Annotated[Model, Field(discriminator="source_type")]
+NodeClasses = t.Union[Model, StandaloneAudit]
```

Comment:
Let's not use `Classes` as part of the type name. To me it feels like it's a list of type **instances** not a type. I suggest perhaps following the `Node` (for union) vs `_Node` (for base class) like we do everywhere else.

Reply:
Seems like it's only used in the next line. Can we just inline?